tests/traced_data/test_io.py

```diff
@@ -90,3 +90,29 @@ def _overwrite_true_asserts(self):
 
         for x, y in zip(data, expected_data):
             self.assertDictEqual(dict(x.items()), y)
+
+
+class TestTracedDataJsonIO(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
+
+    def test_export_traced_data_iterable_to_json(self):
+        file_path = path.join(self.test_dir, "json_test.json")
+        data = generate_traced_data_frame()
+
+        with open(file_path, "wb") as f:
```

Comment:
As discussed - I'd like to be explicit about what is being achieved by the 'b' here